lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -59,10 +59,9 @@
           lang="${course_overview.language}"
         % endif
       >
-<article class="course${mode_class}">
+<section class="course${mode_class}" aria-labelledby="course-title-${enrollment.course_id}">
```

Comment:
@cptvitamin 

The a11y tests are failing for this `section` with error message "Message: <section> elements should have a heading as its first child"

Could you shed some light on what this error means?

Reply:
@BenjiLee the HTML5 spec recommends that a ```section``` element have a heading element as a child.  This one does (The course name), so feel free to ignore this rule

Reply:
@cptvitamin Is there a way to ignore this rule for the tests? The a11y tests are failing in CI.